src/mappings/mappingHandlers.ts

```diff
@@ -260,12 +272,20 @@ export async function handleCw20BalanceTransfer(event: CosmosEvent): Promise<voi
   await saveCw20BalanceEvent(`${id}-debit`, fromAddress, BigInt(0)-BigInt(amount), contract, event);
 }
 
-export async function handleGovProposalVote(msg: CosmosMessage<GovProposalVoteMsg>): Promise<void> {
+export async function handleGovProposalVote(event: CosmosEvent): Promise<void> {
+  const msg: CosmosMessage<GovProposalVoteMsg> = event.msg;
   logger.info(`[handleGovProposalVote] (tx ${msg.tx.hash}): indexing GovProposalVote ${messageId(msg)}`)
   logger.debug(`[handleGovProposalVote] (msg.msg): ${JSON.stringify(msg.msg, null, 2)}`)
```

Comment:
nit: `event.msg.msg` for consistency

```suggestion
  logger.debug(`[handleGovProposalVote] (event.msg.msg): ${JSON.stringify(msg.msg, null, 2)}`)
```